src/mappers/AppUserMapper.ts

```diff
@@ -31,13 +32,13 @@ export class AppUserMapper {
    * Updates an existing AppUser entity with new data from the request payload.
    * If there is no AppUser entity with the passed-in appUserId, then return undefined.
    *
-   * @param {AppUserEventRequest} appUserRequest The request payload from which the updated
+   * @param {AppUserPostRequest | AppUserEventRequest} appUserRequest The request payload from which the updated
    *                                             existing AppUser entity is created.
    * @param {number} appUserId The supposed ID of an existing AppUser entity.
    * @returns {Promise} An updated AppUser entity.
    */
   async requestToExistingEntity(
-    appUserRequest: AppUserEventRequest,
+    appUserRequest: AppUserPostRequest | AppUserEventRequest,
```

Comment:
AppUserPostRequest is unclear, what exactly is this?

Reply:
So I use that for updating and creating an AppUser. Should we just have two separate but with identical content payloads then?

Reply:
🤷‍♂️ was just wondering

Reply:
Oh nah I wasn't being sassy or anything haha. That was my rationale when I named it like that at least

Reply:
oh no no i was just expressing genuinely that idk the answer to your question

Reply:
damn i see why open source contributors get yelled at so much now; tone really doesn't translate well in gh comments

Reply:
it's up to you - I just wanted to point out that it's not immediately clear what a PostRequest is :)

Reply:
I can change AppUserPostRequest to AppUserCreateUpdateRequest for more clarity

Reply:
hmm it's more clearn but then that's kinda long rip

Reply:
maybe we can stick with post then idk, or we could maybe call it base?

Reply:
Idk, they're only used for post requests so I'm not sure if base is the best name